internal/checker/cloudformation/rules/encryption/verify_codebuild_project_encryption.rego

```diff
@@ -0,0 +1,50 @@
+# METADATA
+# title: "Verify that CodeBuild Project encryption is not disabled"
+# description: "Checks whether encryption is enabled for a CodeBuild Project."
+# scope: package
+# related_resources:
+# - https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-codebuild-project.html
+# custom:
+#   id: CB_CFT_061
+#   severity: MEDIUM
+package lib.cloudformation.CB_CFT_061
+
+import future.keywords.in
+
+resource := "AWS::CodeBuild::Project"
+
+is_valid {
+	some resources in input.Resources
+	resources.Type == resource
+}
+
```

Comment:
swap logic for pass and failed then check
encryption.EncryptionDisabled == true